src/magic_folder/cli.py

```diff
@@ -898,96 +873,6 @@ def list_directory(self, uri):
             in dirinfo[u"children"].items()
         })
 
-    @inline_callbacks
-    def download_best_version(self, filenode_uri, progress):
-        uri = self.node_uri.child(
-            u"uri",
-            filenode_uri.to_string().decode("ascii"),
-        ).to_uri().to_text().encode("ascii")
-
-        with start_action(action_type=u"magic-folder:cli:download", uri=uri):
-            response = yield self.agent.request(
-                b"GET",
-                uri,
-            )
-            if response.code != 200:
-                raise Exception(
-                    "Error response from download endpoint: {code} {phrase}".format(
-                        **vars(response)
-                    ))
-
-        returnValue((yield readBody(response)))
```

Comment:
`download_best_version` seems like a clear duplicate of `magic_folder.tahoe_client.TahoeClient.download_capability` so deleting this one seems good (unfortunate that we implemented this (at least) twice).